source/getting-started.txt

```diff
@@ -0,0 +1,63 @@
+.. _app-services-get-started:
+
+================
+Getting Started
+================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Template Apps </template-apps>
+   Tutorials </tutorials>
+
+The resources on this page are designed to help you explore Atlas App 
+Services. Visit the Template Apps page to bootstrap an App Services app. 
+Check out tutorials to learn more about features, such as Device Sync 
+and client authentication.
+
+Template Apps
+-------------
+
+Bootstrap App Services with a Template App. Template apps bring together
+many of the building blocks available in Atlas App Services and start you 
+off with a prebuilt app that you can customize. `Learn more about the Template
+Apps here. <https://www.mongodb.com/docs/atlas/app-services/template-apps/>`_
+
+
+Tutorials
+---------
+
+Follow along with a tutorial to learn how to use App Services.
```

Comment:
It might be good to emphasize that tutorials provide a more guided development path. Maybe something like:

> Unlike Template Apps, which give you a pre-built app to explore and leave you to it, Tutorials provide a detailed step-by-step guide to developing apps that use App Services features.